benchmarks/Tests/Tests.swift

```diff
@@ -0,0 +1,48 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftCertificates open source project
+//
+// Copyright (c) 2023 Apple Inc. and the SwiftCertificates project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftCertificates project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import Sources
+import XCTest
+
+final class TestRunner: XCTestCase {
+    override func setUpWithError() throws {
+        #if DEBUG
+        throw XCTSkip("performance tests only run in release mode")
+        #endif
```

Comment:
Maybe we even want to `fatalError` here?

Reply:
yeah, lets do this.